atc/db/check_factory_test.go

```diff
@@ -348,7 +350,7 @@ var _ = Describe("CheckFactory", func() {
 					Expect(fakeResourceType.CheckPlanCallCount()).To(Equal(1))
 					_, types, version, interval, defaults, _, _ := fakeResourceType.CheckPlanArgsForCall(0)
 					Expect(version).To(Equal(atc.Version{"from": "version"}))
-					Expect(interval.Interval).To(Equal(defaultCheckInterval))
+					Expect(interval.Interval).To(Equal(defaultResourceTypeInterval))
```

Comment:
The test above for check created in DB should also be changed to `defaultResourceTypeInterval`